src/index.css

```diff
@@ -1,14 +1,21 @@
-*,
-*::before,
-*::after {
-  box-sizing: border-box;
-}
 
-body {
+
+:root {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+    sans-serif, "Caveat", cursive, "Rampart One", sans-serif;
```

Comment:
I like that you used some "fancy" fonts. This was the project to do that. Not often that we can go crazy on fun styling and fonts.